server/api/e2e/gql_document_test.go

```
@@ -0,0 +1,466 @@
+					"data": map[string]any{
+						"latestProjectSnapshot": map[string]any{
+							"id":        doc.ID,
+							"updates":   doc.Updates,
+							"version":   doc.Version,
+							"timestamp": doc.Timestamp,
+						},
+					},
+				}
+				w.Header().Set("Content-Type", "application/json")
+				w.WriteHeader(http.StatusOK)
+				json.NewEncoder(w).Encode(resp)
```

golangci-lint (1.62.2)

Error return value of `(*encoding/json.Encoder).Encode` is not checked

(errcheck)